### DISCUSSION TITLE
How to Label and track behaviors inside annolid

### DISCUSSION THREAD

**Discussion by physician1** [healthonrails/annolid#50] [Q&A] [answered]
I am trying to annotate the behaviors and classify them. I am having troble doing that. Below is a procedure from your documentation that I am having trouble understanding.
**For encoding behaviors, please name the ploygon with the behavior name (e.g. rearing, object_investigation, or grooming)**

How can you draw a polygon around a behavior. please explain further for me.

**Comment by healthonrails** [comment, marked as answer]
You have the option to draw a polygon around an object, such as a mouse. When labeling the polygon, you can use descriptors like 'rearing' when the mouse is upright with its head facing upwards. For activities like digging, you can name polygons in specific frames as 'digging,' indicating the instance when the digging occurs between a designated starting and ending frame. I hope this explanation is clear to you.
